skopt/optimizer/base.py

```diff
@@ -274,18 +274,39 @@ def base_minimize(func, dimensions, base_estimator,
 
         models.append(gp)
 
-        if acq_optimizer == "sampling":
+        if acq_optimizer == "sampling" or acq_optimizer == "spearmint":
             X = space.transform(space.rvs(n_samples=n_points,
                                           random_state=rng))
             values = _gaussian_acquisition(
                 X=X, model=gp,  y_opt=np.min(yi), acq_func=acq_func,
                 xi=xi, kappa=kappa)
             next_x = X[np.argmin(values)]
 
-        elif acq_optimizer == "lbfgs":
+        if acq_optimizer == "lbfgs" or acq_optimizer == "spearmint":
             best = np.inf
-            x0 = space.transform(space.rvs(n_samples=n_restarts_optimizer,
-                                           random_state=rng))
+
+            if acq_optimizer == "lbfgs":
+                x0 = space.transform(space.rvs(n_samples=n_restarts_optimizer,
+                                               random_state=rng))
+            else:
+                std = 1e-3
```

Comment:
Shall we make this a parameter of the method and rename it to `spray_std` or something better?